Codeinwp/otter-blocks
# Add Module shortcut in Editor

Closes #1540 .


### Summary


Added shortcuts for enabling the Module Visibility from the Editor. Instead of clicking on the plus icon for Block Tools, you can enable them to be shown by default. This will save a click for the user.

Things to consider:

- Added a link that points to the Dashboard.
- The tab is opened by default.
- Modules that are disabled from Dashboard will not be present.

### Screenshots 



![image](https://github.com/Codeinwp/otter-blocks/assets/17597852/792cfe7b-f915-42c7-873e-14daf02d06a2)


https://github.com/Codeinwp/otter-blocks/assets/17597852/1b47d47d-80e1-4560-a4cb-604b78647607


----

### Test instructions


- Check that the new section is present.
- If you deactivated a Module, that change is reflected in the Dashboard
- No errors, no broken page.
- Sticky does not have the module disabled options, so it will always be present.

⚠️ In this build the console will show some counting (you can see in the video). This will be for detecting loops that can slow down the website. If you see an abnormal growth (going from 1 to 500 in a second or the count does not stop), please report and post the instance.

ℹ️ The console log can be safely removed after the QA test.



---- 

### Checklist before the final review
